models/src/test/java/com/rudderstack/models/MessageUtilsTest.kt

```diff
@@ -1,22 +1,20 @@
 package com.rudderstack.models
 
-import org.hamcrest.MatcherAssert
 import org.hamcrest.MatcherAssert.assertThat
-import org.hamcrest.Matchers
 import org.hamcrest.Matchers.allOf
 import org.hamcrest.Matchers.hasEntry
 import org.hamcrest.Matchers.hasItems
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
 import org.junit.Test
 
-class MessageUtilsTest{
+class MessageUtilsTest {
```

Comment:
Would it be easy to update the names of the test functions to `given` `when` `then` way?

Reply:
I've updated all the test cases in this class in `given` `when` `then` format.